var/spack/repos/builtin/packages/rose/package.py

```diff
@@ -29,19 +29,19 @@
 from spack import *
 
 
-class Rose(Package):
+class Rose(AutotoolsPackage):
     """A compiler infrastructure to build source-to-source program
        transformation and analysis tools.
        (Developed at Lawrence Livermore National Lab)"""
 
     homepage = "http://rosecompiler.org/"
-    url = "https://github.com/rose-compiler/rose/archive/v0.9.7.tar.gz"
+    #url = "https://github.com/rose-compiler/rose/archive/v0.9.7.tar.gz"
```

Comment:
Yes, it is less than ideal, but currently, we do not support tarball releases, though we have in the past. We at least version and tag our tested releases that's why we're using commits. We do plan to support tarball releases again in the near future, but to get this moving forward, we have to just use the Git repository and commits. Otherwise, ROSE won't build properly.